token/js/cli/token-test.js

```diff
@@ -168,20 +172,59 @@ export async function mintTo(): Promise<void> {
   assert(accountInfo.amount.toNumber() === 1000);
 }
 
+export async function mintTo2(): Promise<void> {
+  assert(didThrow(testToken.mintTo2, [testMintAuthority, [], 1000, 1]));
```

Comment:
Ah, thanks for the lead on this @jstarry ! They weren't all passing because of the lack of `await` (just a couple were), but they *were* all passing because `func.apply()` was malformed and always throwing!